src/styles/clock.scss

```diff
@@ -0,0 +1,67 @@
+.clock {
+  position: relative;
+  width: 300px;
+  height: 300px;
+  border: 3px solid $minute-color;
+  border-radius: 50%;
+
+  &::before {
+    content: "";
+    position: absolute;
+    display: block;
```

Comment:
You can create a placeholder for these properties because you use them more 2 times in your project